deployment/v3/utils/prop_migrator/knowledge/new-property-with-decent-default-value.csv

```diff
@@ -1623,3 +1623,27 @@ resident-default.properties,mosip.scope.resident.postRequestShareCredWithPartner
 resident-default.properties,resident.template.support-docs-list
 resident-default.properties,resident.identity.info.version
 resident-default.properties,resident.template.email.content.failure.cust-and-down-my-card
+packet-manager-default.properties,auth.server.admin.allowed.audience
+packet-manager-default.properties,mosip.kernel.idobjectvalidator.referenceValidator
```

Comment:
this could be different based on country, we should remove this config from here

Reply:
removed